webapp/src/language/en.json

```diff
@@ -197,11 +197,15 @@
     "createAccount": "Create Faucet Account",
     "newCreatedAccount": "Your new account is",
     "transferTokensTransaction": "Tokens transferred correctly. Check it on explorer here",
```

Comment:
does this text have a link?

Reply:
Yes, when the faucet is performed, it show a message with the link

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/03bea34c-adce-4329-979a-c3507935f5ff)

https://github.com/edenia/antelope-tools/blob/35efac196af18167def5bd6ac8ade1e3929daf49/webapp/src/routes/Faucet/index.js#L129-L134

Reply:
OK lets use this JSON for the english texts please: 
```
"faucetRoute": {
    "createAccount": "Create Your Blockchain Account",
    "newCreatedAccount": "Congratulations! Your new account has been created:",
    "transferTokensTransaction": "Tokens have been successfully transferred. View the transaction on the explorer:",
    "createButton": "Create Your Account",
    "issueTokens": "Distribute Tokens",
    "getTokens": "Receive Tokens",
    "publicKey": "Enter Public Key (Active/Owner)",
    "accountName": "Choose an Account Name",
    "invalidAccount": "Enter a valid account name, please.",
    "emptyFields": "Complete all fields to proceed with account creation.",
    "accountFormat": "Account name should be 12 characters or fewer. Use lowercase letters (a-z) and numbers (1-5) only.",
    "keyFormat": "Enter a valid ECC key. You can generate one using cleos or any compatible wallet."
}
```